src/context-selection/attribute-option-combo-selector-bar-item/use-category-combination.js

```diff
@@ -46,47 +35,60 @@ const isOptionWithinPeriod = ({
 export default function useCategoryCombination() {
     const [periodId] = usePeriodId()
     const [dataSetId] = useDataSetId()
-    const queryKey = [QUERY_CATEGORY_COMBINATION, { id: dataSetId }]
+    const params = {
+        fields: [
+            'categoryCombo[isDefault,displayName, categories[id, displayName, categoryOptions[id, displayName]]]',
```

Comment:
We're missing startDate and endData here. Is that correct?

Reply:
Yes

Reply:
Yes as in it's ok for these to be removed?

Reply:
No, they are required (until we have the new endpoint)